app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    try:
+        cp, current_file, copied_file = command.split()
```

Comment:
cp here is bad naming because maybe someone enters another value.

Reply:
Also you need to validate command, if here more than 3 splitted values: cp file1.txt file2.txt file2.txt

Reply:
that's why I use the try command. If there are more than three values, a ValueError will appear

Reply:
Oh sorry, my bad

Reply:
Just rename cp

Reply:
but why is it worth renaming cp, because in this particular case it should be the cp command and then I even check it

Reply:
the previous reviewer even advised me to write this way

Reply:
You can say that is 'cp' after checking, before check it just unknown command)
Better to use command or input_command.